README.md

```diff
@@ -1,2 +1,136 @@
+
+Requirements: One voucher from any user with level 2 or above access.
+
+Contributors with Triage access will have the associated [triage Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Program Library](https://github.com/solana-labs/solana-program-library) (SPL) and the
+[Solana](https://github.com/solana-labs/solana) repositories.
+
+### Level 2 - Write Access
+
+Requirements: Two voucher from anyone with level 3 access or two vouchers from
+anyone from level 2. Applicants must have level 1 access to apply for level 2.
+
+Contributors with Write access will have the associated [write Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Program Library](https://github.com/solana-labs/solana-program-library) (SPL) and the 
+[Solana](https://github.com/solana-labs/solana) repositories.
+
+Most notably these permissions allow the contributor to approve and merge pull
+requests and gets them access to Solana's Buildkite CI.
+
+### Level 3 - Maintain Access
+
+Requirements: Two voucher from anyone with level 3 access.
+
+Contributors will have the associated [maintain Github access
+policy](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)
+for the [Solana Program Library](https://github.com/solana-labs/solana-program-library) (SPL) and the 
+[Solana](https://github.com/solana-labs/solana) repositories.
+
+Users with Maintain access are responsible for managing the the repositories and
+granting user access based on the above requirements.
+
+### Vouching Process 
+
+To receive a Contributor status and the respective level of access as defined in the Contributor Access Policy, follow these steps:
+
+1. Open an issue on the [Contributor Access Policy](https://github.com/solana-labs/contributor-access-policy) repository
+with a title in the following format: "Request Access (Level [X])
+for [Username]". Add information that can help identify you such as your 
+Discord ID in the issue.
+2. Gather your vouchers to add a comment on the issue expressing their support.
+3. Once the issue has received enough support, notify the users with Maintain
+access by adding a comment tagging
+[`@admins`](https://github.com/orgs/solana-labs/teams/admins)
+and you will be granted the requested level of access.
+4. It would take 3 business days for the request to be processed.
+
+To receive a Core Contributor status and the respective level of access as defined in the Contributor Access Policy, follow these steps:
+
+1. Open an issue on the [Contributor Access Policy](https://github.com/solana-labs/contributor-access-policy) repository
```

Comment:
Make them link to their previously approved Contributor access issue as well. So you can't just jump right to Core Contrib